test/zip-dir.test.js

```diff
@@ -99,6 +100,86 @@ describe("zip-dir", function () {
     });
   });
 
+  describe("Ensures that inner folder names can be defined", function () {
+    afterEach(cleanUp);
+
+    function customCompareMethod(file, customFilePath = "", combineCustomFilePathOrReplace = true) {
+      const zipBuffer = fs.readFileSync(path.join(sampleZipPath, file));
+      const combinedFilePath = path.join(customFilePath, file);
+      // Compare the different file buffers.
+      const fileBuffer = fs.readFileSync(
+        path.join(
+          outputPath,
+          combineCustomFilePathOrReplace ? combinedFilePath : customFilePath
+        )
+      );
+
+      expect(bufferEqual(zipBuffer, fileBuffer)).to.be.ok;
+    }
```

Comment:
I can move the this function to the global def, but thought i would ask before changing the other aspects. :)